ocs_ci/ocs/bucket_utils.py

```diff
@@ -3191,3 +3258,34 @@ def _check_objs_deletion():
         result=True
     ), "Not all the objects are marked deleted"
     logger.info("All the objects are marked expired")
+
+
+def verify_deletion_marker(mcg_obj, awscli_pod, bucket_name, object_key):
+    """
+    Verify if deletion marker exists for the given object key
```

Comment:
It's possible to have a delete marker with IsLatest=false:
```
$ my_s3 api put-bucket-versioning --bucket first.bucket --versioning-configuration Status=Enabled
$ echo 'a0' | my_s3 cp - s3://first.bucket/a
$ echo 'a1' | my_s3 cp - s3://first.bucket/a
$ my_s3 rm s3://first.bucket/a
delete: s3://first.bucket/a
$ echo 'a2' | my_s3 cp - s3://first.bucket/a
$ my_s3 api list-object-versions --bucket first.bucket | jq .DeleteMarkers
[
  {
    "Owner": {
      "DisplayName": "admin@noobaa.io",
      "ID": "67ebd24eaac453001d17838d"
    },
    "Key": "a",
    "VersionId": "nbver-3",
    "IsLatest": false,
    "LastModified": "2025-04-02T14:00:57+00:00"
  }
]
```

So please refine the method's name and docstring accordingly:
```suggestion
def verify_soft_deletion(mcg_obj, awscli_pod, bucket_name, object_key):
    """
    Verify if deletion marker exists and IsLatest for the given object key
```

Reply:
@sagihirshfeld sure. why do you call it `soft deletion`?